addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -261,8 +283,28 @@ odoo.define('website.s_website_form', function (require) {
 
             // Prepare form inputs
             this.form_fields = this.$target.serializeArray();
+
+            let tooManyFiles = false;
+            let fileTooBig = false;
```

Comment:
Do you need 2 variables here ? Only 1 variable ("notValid" or something like that) is not enough ?